app/components/nav-rail-pinned-item.tsx

```
@@ -28,17 +28,26 @@ export default function NavRailPinnedItem(
-        <div className={`nav-item-common flex items-center rounded-full cursor-pointer font-medium selected-${props.selected}`} role='link' onClick={props.onClick}>
-            <Link className={`state-layer relative flex extended-gap-${props.extended} items-center justify-between rounded-full ${!props.extended ? 'max-w-6' : ""} padding-${props.extended ? "extended" : "collapsed"}`} href={props.href!}>
-                <md-ripple />
-                <div className={`nav-item-content flex extended-gap-${props.extended} items-center`}>
-                    <div className="cover flex items-center">
-                        <Image src={props.imgSrc} alt={props.text} width={props.width} height={props.width} />
-                    </div>
-                    <div className={`label-container text-ellipsis items-center extended-${props.extended}`}>
-                        <p className={`label text-center text-ellipsis extended-${props.extended}`}>{props.text}</p>
-                    </div>
-                </div>
+        <div className={`nav-item rounded-full cursor-pointer font-medium selected-${props.selected}`} role='link' onClick={props.onClick}>
```

Biome

[error] The HTML element with the interactive role 'link' is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like 'link'.
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)